### ISSUE TITLE
Continue has been slow down

### ISSUE THREAD

**Issue by nothingness6** [continuedev/continue#1736]
### Validations

- [X] I believe this is a way to improve. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [X] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement) that requests the same enhancement

### Problem

Since I updated Continue last time, it has been slow down drastically. Every time I copy and paste the code, it takes pretty long time compared to the previous version. It was blazing fast though. 
Hope it'll be fixed in the next update. 
Thanks!

**Comment by Patrick-Erichsen** [comment]
HI @nothingness6 . would you mind sharing the following info?

```
- Your operating system:
- What version of Continue you're currently using:
- What IDE you're using (e.g. VS Code or IntellijJ:
```

Please share anything else that might be relevant, e.g. if you are running over SSH.

**Comment by jonnyboynewton** [comment]
Is this the same as #1705 / #1771 ?

**Comment by spew** [comment]
Duplicate of https://github.com/continuedev/continue/issues/1774 as well

**Comment by abishekmuthian** [comment]
I face the issue only when using Codestral model with ollama provider, which was working fine earlier(unable to find last working version). Codestral via Continue is 4 times slower than when using Codestral with open-webui. I've now switched to using deepseek-coder-v2 until the issue is resolved.

VSCode
Continue: 0.8.43
Nvidia 4090 Mobile (16GB VRAM)
Linux 6.9.x

**Comment by spew** [comment]
Can you retest, we have made extensiving indexing performance fixes and there is the fix for https://github.com/continuedev/continue/issues/1908

**Comment by nothingness6** [comment]
> HI @nothingness6 . would you mind sharing the following info?
> 
> ```
> - Your operating system:
> - What version of Continue you're currently using:
> - What IDE you're using (e.g. VS Code or IntellijJ:
> ```
> 
> Please share anything else that might be relevant, e.g. if you are running over SSH.

My apologies for the late response. Here is the info you requested :
> - Your operating system: Windows 10 64 bit 22H2
> - What version of Continue you're currently using: The latest version 
> - What IDE you're using (e.g. VS Code or IntellijJ: VS Code

**Comment by Kakiho** [comment]
I have the same problem!
I write on TypeScript and when I paste the code, it is inserted only after 3-7 seconds.

At the same time, the tsserver loads the processor, and in the editor, on the line where I paste, a loading animation appears.

Env:
System: Zorin OS 17.1 (Ubuntu)
VSCode: 1.94.2
Continue: 0.8.55
